components/admin_console/admin_definition.jsx

```diff
@@ -2972,6 +2972,369 @@ export default {
+                            key: 'ServiceSettings.ExperimentalEnableHardenedMode',
+                            label: t('admin.experimental.experimentalEnableHardenedMode.title'),
+                            label_default: 'Enable Hardened Mode',
+                            help_text: t('admin.experimental.experimentalEnableHardenedMode.desc'),
+                            help_text_default: 'Enables a hardened mode for Mattermost that makes user experience trade-offs in the interest of security. See [documentation](!https://docs.mattermost.com/administration/config-settings.html#enable-hardened-mode-experimental) to learn more.',
+                            help_text_markdown: true,
+                        },
+                        // {
+                        //     type: Constants.SettingsTypes.TYPE_BOOL,
+                        //     key: 'ServiceSettings.ExperimentalStrictCSRFEnforcement',
+                        //     label: t('admin.experimental.experimentalStrictCSRFEnforcement.title'),
+                        //     label_default: 'TODO',
+                        //     help_text: t('admin.experimental.experimentalStrictCSRFEnforcement.desc'),
+                        //     help_text_default: 'TODO',
+                        //     help_text_markdown: false,
+                        // },
+                        {
+                            type: Constants.SettingsTypes.TYPE_BOOL,
+                            key: 'TeamSettings.EnableUserDeactivation',
+                            label: t('admin.experimental.enableUserDeactivation.title'),
+                            label_default: 'Enable Account Deactivation',
+                            help_text: t('admin.experimental.enableUserDeactivation.desc'),
+                            help_text_default: 'When true, users may deactivate their own account from **Account Settings > Advanced**. If a user deactivates their own account, they will get an email notification confirming they were deactivated. When false, users may not deactivate their own account.',
+                            help_text_markdown: true,
+                        },
+                        {
+                            type: Constants.SettingsTypes.TYPE_BOOL,
+                            key: 'TeamSettings.EnableXToLeaveChannelsFromLHS',
+                            label: t('admin.experimental.enableXToLeaveChannelsFromLHS.title'),
+                            label_default: 'Enable X to Leave Channels from Left-Hand Sidebar',
+                            help_text: t('admin.experimental.enableXToLeaveChannelsFromLHS.desc'),
+                            help_text_default: 'When true, users can leave Public and Private Channels by clicking the “x” beside the channel name. When false, users must use the **Leave Channel** option from the channel menu to leave channels.',
+                            help_text_markdown: true,
+                        },
+
+                        // {
```

Comment:
The server needs an actual array (vs. a comma-separated list), so I'm deferring this one for now.